DemoServer/Controllers/Demos/StaticIndexes/StoreFieldsOnIndex/metadata.json

```diff
@@ -0,0 +1,248 @@
+            "Title": "Query for documents",
+            "Slug" : "step-6",
+            "DescriptionLines": [
+                "* Query the 'Orders' collection using the defined index.",
+                "",
+                "* The orders' documents can be _filtered_ by the index-fields defined in `OrdersQuantity_ByCompany.IndexEntry`",
+                "",
+                "* The resulting documents will include only orders that where made by 'companyId'."
+            ],
+            "Assets": [
+
+                {
+                    "Title": "Filtering Results - Basics Demo",
+                    "Type" : "demo",
+                    "Url"  : "/queries/filtering-results-basics"
+                }
+            ]
+        },
+        {
+            "Title": "Project Results",
+            "Slug" : "step-7",
+            "DescriptionLines": [
+                "* Use `ProjectInto()` to make the server return only the fields defined in the 'OrdersQuantity_ByCompany.OrderProjectedDetails' class per document result instead of the whole document.<br>",
+                "  * If the field is _stored on the index_ - the server will fetch it from the index<br>",
+                "  * If the field is _not stored on the index_ - the server will fetch it from the document store",
+                "",
+                "* In this example:<br>",
+                "  * 'TotalItemsOrdered' is fetched from the index<br>",
+                "  *  'OrderedAt' if fetched from the document store",
+                "",
+                "* Method _Select()_ can be used instead of ProjectInfo(). See related links."
+            ],
+            "Assets": [
+                {
+                    "Title": "Project Index Result Demo",
+                    "Type" : "demo",
+                    "Url"  : "/static-indexes/project-index-results"
+                },
+                {
+                    "Title": "Querying: Projections",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/Csharp/indexes/querying/projections"
+                },
+                {
+                    "Title": "Select()",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/indexes/querying/projections#select"
+                }
+            ]
+        },
+        {
+            "Title": "Execute the Query",
+            "Slug" : "step-8",
+            "DescriptionLines": [
+                "* The _ordersQuery_ variable only stores the query definition.<br>",
+                "  Execute the query by calling `ToList()`, or any other method that forces immediate execution.",
+                "",
+                "* The final query results include the projected details for orders that where made by 'companyId'.",
+                "",
+                "* The resulting objects are Not tracked by the session since they are not the stored documents but only partial data."
```

Comment:
`Not tracked` - is this a typo or intended emphasizing of the fact that there is no tracking

Reply:
Resolving since now I understand this is a subtle emphasis